documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -6,15 +6,15 @@
 [id='assembly-upgrade-{context}']
 = Upgrading Strimzi
 
-Strimzi can be upgraded with no cluster downtime.
+Strimzi can be upgraded without causing any downtime for consumers and producers. 
+Cluster availability might be affected by rolling updates that occur during the upgrade process.
+
 Each version of Strimzi supports one or more versions of Apache Kafka.
 You can upgrade to a higher Kafka version as long as it is supported by your version of Strimzi.
 In some cases, you can also downgrade to a lower supported Kafka version.
 
 Newer versions of Strimzi may support newer versions of Kafka, but you need to upgrade Strimzi _before_ you can upgrade to a higher supported Kafka version.
```

Comment:
Not strictly needed anymore with the new upgrades. So maybe we can leave this sentence out?

Reply:
Not done. I'd rather change this in a subsequent PR for multi-version upgrades.